src/safeposix/syscalls/net_calls.rs

```diff
@@ -1859,6 +1859,38 @@ impl Cage {
         }
     }
 
+    //------------------------------------SELECT SYSCALL------------------------------------
```

Comment:
Needs some minor format changes to work well with Rustdoc.  See @yashaswi2000 's PR for tips...

Reply:
Thanks, just updated the comments